server/src/main/java/org/opensearch/search/aggregations/bucket/composite/CompositeAggregator.java

```diff
@@ -97,7 +97,7 @@
  *
  * @opensearch.internal
  */
-final class CompositeAggregator extends BucketsAggregator {
+public final class CompositeAggregator extends BucketsAggregator {
```

Comment:
It looks like you're making this class public in order to access `CompositeAggregator.CompositeAggregationType`. I'm wondering if there's a better way to do that. For example, you could add a `getSize()` method to `AggregationType` so you don't need to cast.

Reply:
make sense, changed